scripts/run-tests.js

```diff
@@ -21,14 +21,14 @@ var jsfiles = common.expand([pwd() + '/*.js',
                              pwd() + '/src/*.js',
                              pwd() + '/test/*.js'
                             ]).join(' ');
+// Perform linting on all javascript files
 if (exec(JSON.stringify(process.execPath)+' '+pwd()+'/'+JSHINT_BIN+' '+jsfiles).code !== 0) {
   failed = true;
   echo('*** JSHINT FAILED! (return code != 0)');
-  echo();
 } else {
   echo('All JSHint tests passed');
-  echo();
 }
+echo();
```

Comment:
Just moving this `echo()` call out of the if-else, since we do it for either condition.